test/tests/performance/pbs_cgroups_stress.py

```diff
@@ -131,25 +132,37 @@ def setUp(self):
         # Restart mom so exechost_startup hook is run
         self.mom.signal('-HUP')
 
-    @staticmethod
     def get_paths():
         """
         Returns a dictionary containing the location where each cgroup
         is mounted.
         """
-        paths = {}
+        paths = {'pids': None,
+                 'blkio': None,
+                 'systemd': None,
+                 'cpuset': None,
+                 'memory': None,
+                 'memsw': None,
+                 'cpuacct': None,
+                 'devices': None}
```

Comment:
The changes you made to get_paths() are fine, but you could have also borrowed _get_paths() and _assemble_path() from the cgroup hook itself.

Reply:
Thanks Mike. I wanted it to be similar to TestCgroupsHook under functional category.